### DISCUSSION TITLE
What is the solution to pass environment variables to the web bundle?

### DISCUSSION THREAD

**Discussion by iuliiadefault** [necolas/react-native-web#2158] [Q&A] [answered]
Is there any ways to pass .env variables to the bundle? react-native-config and react-native-dotenv seem to not working.

**Comment by intergalacticspacehighway** [comment, marked as answer]
React native config does not support web yet and needs some workaround https://github.com/luggit/react-native-config/issues/225. 
react-native-dotenv should work though. Did you check this issue? https://github.com/goatandsheep/react-native-dotenv/issues/91 Also, can you confirm the babel setup as mentioned here? https://github.com/goatandsheep/react-native-dotenv#usage

**Reply by iuliiadefault** [reply]
Just switched to the `react-native-dotenv`, thanks!
